Sources/SwiftProtobuf/Message.swift

```diff
@@ -31,7 +31,7 @@
 ///
 /// The actual functionality is implemented either in the generated code or in
 /// default implementations of the below methods and properties.
-public protocol Message: _CommonMessageConformances {
+public protocol Message: _CommonMessageConformances, PopulatorConstructable {
```

Comment:
You might want to reference back to https://github.com/apple/swift-protobuf/pull/1498 and the things that references, I'm not sure if this would have size impacts we don't want or not.